fuzz-tests/src/test/java/org/roaringbitmap/Fuzzer.java

```diff
@@ -148,7 +148,14 @@ public static void verifyInvariance(String testName, int maxKeys, Predicate<Roar
               RoaringBitmap two = randomBitmap(maxKeys);
               if (validity.test(one, two)) {
                 try {
-                  Assert.assertEquals(left.apply(one, two), right.apply(one, two));
+                  T res1 = left.apply(one, two);
+                  T res2 = right.apply(one, two);
+                  if (!res1.equals(res2)) {
```

Comment:
I'm not sure I understand the rationale for this change. Is this intentional? It's assumed that `left` is pure. If these isn't a good reason could you revert this please?

Reply:
Oops, I changed that code for debugging purposes. I forgot to reset the change. reverted.

Reply:
This isn’t resolved- the res1.equals(res2) line below also looks like debugging code. Can we revert this prior to merging please? 

FYI @lemire

Reply:
@richardstartin, indeed, I lost the revert while pushing. should be ok now.